docs/release-notes.rst

```diff
@@ -1,3 +1,289 @@
+
+**Fix:** This has been fixed in APIC versions 4.2(4p)+ and 5.0(2e)+.
+
+**Workaround:** To download visibility reports, use a different browser, like Mozilla Firefox, or an earlier version of Google Chrome.
+
+------
+
+**Visibility table fast scrolling on Mozilla Firefox for scale configs may result in a blank screen**
+
+This is a known ag-grid issue on the Mozilla Firefox browser: https://github.com/ag-grid/ag-grid/issues/2841
+
+**Workaround:** Scroll slowly to prevent this issue. But if this issue is observed, the possible workarounds are:
+
+1. Switch tabs from the Visibility tab to one of the other tabs and then switch back. 
+
+2. Select a different entry from the Visibility table drop-downs (either Partition or Table) and switch back to the intended combination.
+
+3. Close and re-open the FASC app UI.
+
+------
+
+**Visibility table status icons render few seconds after data for scale configs**
+
+**Workaround:** None
+
+------
+
+**BIG-IP v12 displays BIG-IP logs in local timezone**
+
+**Visibility Dashboard → View Logs** (for both VIP and Node) will display logs in UTC. But for BIG-IP v12, the logs are displayed in local timezone (timezone of the BIG-IP).
+
+**Workaround:** Upgrade BIG-IP to v13 or higher.
+
+------
+
+**Visibility Dashboard filter may display additional logs with interface filter**
+
+**Visibility Dashboard → BIG-IP Endpoint Details → Interface → View Logs** (for BIG-IP interfaces) displays interface logs. By default it applies the filter of interface name (for example: 1.1 or 1.2); and hence only the logs with interface name in them are displayed. This filtering logic may not work as expected and display additional logs which have interface names as a substring (For example: **1.2** is a substring in the log "Pool /Partition/Application/web_pool member **/Partition/12.14.1.2:8080** monitor status down").
+
+**Workaround:** None.
+
+------
+
+L4-L7 App Services
+------------------
+
+**Application services declaration not deleted**
+
+If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, the Application Services configuration may not be deleted completely, even after multiple attempts from the application UI. However, the configuration is removed from the BIG-IP device.
+
+**Workaround**: Upload an AS3 sample declaration to the app and then perform a :guilabel:`Delete all` operation. (Use :guilabel:`View AS3 Declaration` and click :guilabel:`Delete`.)
+
+-------
+
+**Once all Pool Member cards are removed from the template form, adding a new pool member card fails on FASC FAST templates UI**
+
+If a user clicks the "-" sign in front of the pool members card and tries to add pool member card again by clicking the "+" icon, then the pool member card is not populated in template form.
```

Comment:
... pool members card, and then tries to add the pool member card again by clicking the "+" icon, the pool member card is not...